release_automation.sh

```diff
@@ -346,8 +346,9 @@ execute "git" "push" "-u" "origin" "HEAD"
 ohai "Create release branch in WordPress-iOS"
 execute "git" "switch" "-c" "gutenberg/integrate_release_$VERSION_NUMBER" 
 
-ohai "Update gutenberg-mobile ref"
+ohai "Update GitHub organization and gutenberg-mobile ref"
 test -f "Podfile" || abort "Error: Could not find Podfile"
+sed -i'.orig' -E "s/wordpress-mobile(\/gutenberg-mobile)/$MOBILE_REPO\1/" Podfile || abort "Error: Failed updating GitHub organization in Podfile"
```

Comment:
Just curios what is the purpose of `\1` working here?

Reply:
The regex here contains a capture group matching the string `/gutenberg-mobile`. The `\1` outputs the contents of the capture group into the replacement string. I.e. it ensures `/gutenberg-mobile` remains in the line.